addons/binding/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/DishwasherChannelSelector.java

```diff
@@ -45,56 +46,56 @@
         @Override
         public State getState(String s, DeviceMetaData dmd) {
             Date date = new Date();
-            SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
-            DATE_FORMATTER.setTimeZone(TimeZone.getTimeZone("GMT+0"));
+            SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
```

Comment:
Should we really keep the `SimpleDateFormat`?

Reply:
I decided to just fix the SAT warnings to reduce the risk of regressions since the binding has no test coverage and I can't test it myself.

Reply:
Okay, makes sense.